src/main/java/org/example/servletstexts/dao/TextsDAO.java

```diff
@@ -0,0 +1,35 @@
+package org.example.servletstexts.dao;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class TextsDAO {
+
+    private int currentId = 0;
+    private final Map<Integer, String> texts = new HashMap<>();
```

Comment:
Customer told us that expecting 10+ billion texts created during next 5 years.
Which problem do you see in this case ?